### DISCUSSION TITLE
custom hostgroup/portgroup

### DISCUSSION THREAD

**Discussion by securityfai** [Security-Onion-Solutions/securityonion#12845] [2.4]
### Version

2.4.50

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

4

### RAM

16

### Storage for /

200

### Storage for /nsm

200

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

In the SOC, we are allowed to create up to 9 custom hostgroup and custom portgroup to allow a set of custom hosts to connect to a custom port on a specific node, how can we proceed if this number is exhausted

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by Sahale** [comment]
We are waiting for it too #12429
